pkg/commands/process/settings/rules/javascript/express/path_traversal.yml

```diff
@@ -8,6 +8,9 @@
           - resolve
       - variable: USER_INPUT
         detection: javascript_express_path_traversal_request_obj
+      - not:
+          variable: USER_INPUT
+          detection: javascript_express_path_traversal_sanitized_user_input
 auxiliary:
   - id: javascript_express_path_traversal_request_obj
     patterns:
```

Comment:
Please clarify the expected safe characters for `replace` calls in the `javascript_express_path_traversal_sanitized_user_input` pattern.

Reply:
Thanks - good idea. I've added some extra information to the `remediation` copy that provides an example of the expected safe characters here https://github.com/Bearer/bearer/pull/759/commits/8b7e363741620ad681db772a922dc46df6d2cda2